lib/cadet/model_helper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Cadet.ModelHelper do
```

Comment:
Try to move all the JSON deserialising in the models here for more DRY-ness (Don't Repeat Yourself)

Reply:
Bump